### ISSUE TITLE
Uselessly force a lang attribute on the whole document.

### ISSUE THREAD

**Issue by Mbodin** [validator/validator#739]
My webpage is written in several languages. I made sure that each textual element is enclosed in a `<div>` element with the appropriate `xml:lang = "fr" lang = "fr"` attributes. There is thus no need to add a dummy language for the whole page, as the language of each text on the page is already specified, and that the page has no main language. If not empty, such a declared global language may even make browsers make false assumptions on the page. Yet the validator issues the following warning:
```
This document appears to be written in English. Consider adding lang="en" (or variant) to the html start tag.
```
I think that it would be nice to weaken this warning, and to only make it appear if there effectively is a text of undeclared language in the page. I would also argue that the warning should not be placed at the global `<html>`, but at the place where the text with no language is used.
From my understanding of the standard, users are only encouraged to put a global language precisely to avoid having texts of unknown language in the page. But if the user already made sure that none appear, this warning doesn’t really make sense.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/739#issuecomment-446386621. Please comment further if there’s any remaining problem
